script/countries.py

```diff
@@ -0,0 +1,15 @@
+"""Helper script to update country list.
+
+ISO does not publish a machine readable list free of charge, so the list is generated
+with help of the pycountry package.
+"""
+import pycountry
+
+COUNTRIES = sorted({x.alpha_2 for x in pycountry.countries})
+
+print(
+    '    {\n        "',
+    '", \n        "'.join(COUNTRIES),
+    '",\n    }',
+    sep="",
+)
```

Comment:
Hm - instead of formatting this by hand here, I'd probably just pipe some Python through `black`? There's precedent for that here:
https://github.com/home-assistant/core/blob/1021c90bb8723fc827efd49abd472a8817d1ec9a/script/hassfest/mqtt.py#L40

Reply:
You're right, and I even added the call to black myself 🙈